choose_license.qmd

```diff
@@ -29,6 +29,12 @@ You can adapt the following wording to your use case:
 
 Whenever you create a literary or artistic work (such as a text, image, video, or software), the copyright law in most countries limits other people from copying, modifying, and sharing it without your express permission. [If the work was created as part of your job, it might be your employer who holds the copyright to the work, depending on the country and contract.]{.aside} This even applies if you make it available to others (e.g., on your website): First and foremost they are not allowed to copy, modify, or share it. This legal default of "all rights reserved" was created to benefit publishers, not authors [@Fogel2006], and runs counter to many cultural and scientific processes. Copyright licenses enable authors to free up their works for reuse by others.
 
+::: {#nte-situation-lmu .callout-note collapse="true"}
+### For researchers at LMU
+
+If you are an employee at LMU and your funder does not mandate the use of a particular license, the decision which license to use lies with your associated chair. In the (rare) case the software you developed has a technical application and constitutes an invention, you are required to officially report your invention to the university which decides on its release. If the university claims the invention, you may not apply a copyright license that makes a patent grant. More information about this process is provided in the [service portal](https://www.serviceportal.verwaltung.uni-muenchen.de/internationals/research_transfer/intellectual_property/index.html). More information about software patents are provided by the FSFE: [Software Patents in Europe](https://fsfe.org/activities/swpat/swpat.en.html).
```

Comment:
Noting the rarity of needing higher-level approval is good, but I would also strongly emphasize the other dimension of that--LMU researchers will almost always, with chair permission, be able to license their research projects (i.e. analyses, code, etc.) with whatever FOSS license(s) they choose.

Reply:
I tried to address this in 516145b